### DISCUSSION TITLE
8月5日续费2年，现在变成免费账户了

### DISCUSSION THREAD

**Discussion by zzwushuang** [getlantern/lantern-forum-cn#808] [Q&A]
![局部截取_20240824_154233](https://github.com/user-attachments/assets/168eca61-ee28-454d-a29b-ce1dd152b3b0)
如题，以及支付记录

**Comment by zzwushuang** [comment]
账号是：1012763460@qq.com

**Comment by zzwushuang** [comment]
@Mike777Khoo 续费两年也没人管吗........

**Comment by zzwushuang** [comment]
不行退款吧，消费都整的那么惆怅

**Comment by zzwushuang** [comment]
@Mike777Khoo

**Reply by zzwushuang** [reply]
解决一下，谢谢

**Comment by zzwushuang** [comment]
人呢，工作日也见不到人？

**Reply by MaxMKMax** [reply]
@zzwushuang 您好. 我走美国时间的. 才刚上线呢. 马上帮你们处理.

**Comment by MaxMKMax** [comment]
@zzwushuang 已经帮您提交问题上去处理了. 谢谢您支持蓝灯! 非常抱歉让您周末等了.
